client/src/app.scss

```
@@ -277,3 +277,9 @@ u {
 ._ify {
   @include mdn.uify;
 }
+
+.icon {
+  @media (forced-colors: active) {
```

Oh, and can we swap these?

The idea would be to add more `force-colors`-specific rules here later, if needed:

```suggestion
@media (forced-colors: active) {
  .icon {
```

---

Done